opengrok-web/src/test/java/org/opengrok/web/api/v1/controller/ConfigurationControllerTest.java

```diff
@@ -291,4 +297,70 @@ public void run() {
         // Revert the value back to the default.
         env.setHitsPerPage(origValue);
     }
+
+    @Test
+    public void test() throws InterruptedException, IOException {
+        final int nThreads = 40;
```

Comment:
possibly make the 2 constants based on number of CPUs in the system ?

Reply:
I added the second constant and made the nThreads dependent on the testing system cpu